Person/personattributetype.md

```diff
@@ -0,0 +1,112 @@
+# Person Attribute Type
+
+## Overview
+
+* A PersonAttributeType exists to group PersonAttribute objects together.
+
+## Available operations.
+
+1. [List person attribute types](#list-person-attribute-types)
+2. [Create a person attribute type](#create-a-person-attribute-type)
+3. [Update a person attribute type](#update-a-person-attribute-type)
+4. [Delete a person attribute type](#delete-a-person-attribute-type)
+
+
+### List person attribute types
+
+* #### List all non-retired person attribute types.
+
+    Quickly filter person attribute types with a given search query. Returns a `404 Not Found` status if the person attribute type not exists.
+     If the user not logged in to  perform this action, a `401 Unauthorized` status returned.
+
+    ```console
+    GET /personattributetype
+     ```
+
+* #### List person attribute type by UUID.
+
+    Retrieve a person attribute type by its UUID. Returns a `404 Not Found` status if the person attribute type not exists. If the
+    user not logged in to  perform this action, a `401 Unauthorized` status returned.
+
+    ```console
+    GET /personattributetype/:target_person_attribute_type_uuid
+    ```
+
+### Create a person attribute type
+
+* To Create a person attribute type you need to specify below attributes in the request body.If the user not logged in to perform this action,
+ a `401 Unauthorized` status returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the person attribute type (Required)
+    *description* | `String` | Description (Required)
+    *format* | `java.package.Class` | the Java class the PersonAttributeType  
+    *foreignKey* | `Number` | the foreign key in the database that this PersonAttributeType references
```

Comment:
the internal identifier (foreign key) to a Concept that defines the possible values for this attribute